.github/workflows/swift.yml

```diff
@@ -22,8 +22,6 @@ jobs:
       fail-fast: false
       matrix:
         runs-on:
-          - os: macos-12
```

Comment:
Would we want to still do a bigger matrix on the main? Our matrix should include all OS versions we support officially, otherwise, devs can introduce breaking changes unintentionally.

Reply:
So this is just for building which we only need to do on the latest macOS. In our Xcode config we select the macOS to target for running. For automated testing we'll need another approach with much more targets, like possibly Xcode Cloud.

For building the `gateway` and `relay` binaries, we'll want to do that in Docker anyway to have more control over the `libc` version used in the final executable.